lib/cf/CloudControllerClient.js

```diff
@@ -227,6 +227,14 @@ class CloudControllerClient extends HttpClient {
   getSpaces(options) {
     return this.createSpaceStream(options).all();
   }
+  getSpaceGuid(instance_guid, options) {
```

Comment:
I would not have this wrapper method but there is nothing wrong with it either. Just that you seem to have a wrapper method in basecontroller as well, so generally wrapping a wrapper method does not seem right. You could have directly done the contents of this method in BaseController. Or even better remove the wrapper functions from BaseController and let the admin controller directly refer this method. Either ways look at how wrapper functions are created. I feel it might be better to remove the wrapper methods in BaseController, so that in future all other modules can then leverage the same.